crates/core/migrations/2022-10-17-074109_create_wallet_total_rewards_table/up.sql

```diff
@@ -0,0 +1,42 @@
+create table wallet_total_rewards (
+    id uuid primary key default gen_random_uuid(),
+    wallet_address varchar(48) not null,
+    reward_center_address varchar(48) not null,
+    total_reward numeric not null
+);
+
+create index wallet_total_rewards_wallet_address_idx
+on wallet_total_rewards (wallet_address);
+
+create index wallet_total_rewards_reward_center_address_idx
+on wallet_total_rewards (reward_center_address);
+
+create function update_total_rewards() returns trigger
+    language plpgsql
+    as $$
+begin
+    update wallet_total_rewards set total_reward =
+    case 
+        when wallet_address in
+        (
+            select wallet_address
+            from reward_payouts
+            where wallet_address = new.buyer
+        )
+        then total_reward + new.buyer_reward
+        when wallet_address in
+        (
+            select wallet_address
+            from reward_payouts
+            where wallet_address = new.seller
+        )
+        then total_reward + new.seller_reward
+    end
+    where reward_center_address = new.reward_center;
```

Comment:
Needs to handle creating the record for the buyer or the seller when it doesn't exists